.circleci/config.yml

```diff
@@ -279,6 +279,19 @@ commands:
             - fxa-yarn-002-{{ checksum "yarn.lock" }}
             - fxa-yarn-002-
 
+  cache-save-nx:
+    steps:
+      - save_cache:
+          key: fxa-nx-001-{{ checksum "yarn.lock" }}-{{ .Revision }}
```

Comment:
One thing to watchout for. This cache can get really large, and with this setup, we might be downloading a bunch of irrelevant stuff. The better way to do this is with NX cloud or rolling our cache storage, which would only pull down the parts of the cache that are needed.

Also, just curious, why cache on yarn.lock? Are you thinking this changes enough to avoid a really large cache?